server/py/framework/utils/background_tasks/kinds.py

```diff
@@ -21,3 +21,4 @@ class BackgroundTaskKinds:
     function_deletion = "function.deletion.{0}"
     refresh_smtp_configuration = "smtp.configuration.refresh"
     push_notification = "notification.push.{0}.{1}"
+    push_pipeline_notification = "notification.push.{0}.{1}.{2}"
```

Comment:
lets have some difference between `push_notification` and `push_pipeline_notification`, to more easily differentiate
```suggestion
    push_pipeline_notification = "notification.kfp.push.{0}.{1}.{2}"
```